webapp/tests/forms/flows/test_lotse_flow.py

```diff
@@ -1008,6 +1010,29 @@ def test_if_familienstand_step_then_delete_person_b_data_correctly(self):
             for person_b_field in person_b_fields:
                 self.assertNotIn(person_b_field, returned_data)
 
+    def test_if_familienstand_step_and_familienstand_changes_to_married_then_delete_is_user_account_holder(self):
```

Comment:
Do you think we should add a test to make sure that the us_user_account_holder and account_holder are not deleted if they are already present ? (i.e. if you again post you're married and again post you're single.

Reply:
Basically, you mean testing for the original bug?

Reply:
Yes.